tests/test__data.py

```diff
@@ -169,15 +169,10 @@ def test___getslice__(self):
         exp = [85.0, 90.0, 95.0, 100.0, 105.0] 
         self.assertListEqual(val, exp)            
         
-    def test___init__kwargs(self):
-        # TODO: is this a valid test?  Did anything pass?
-        d = sm.data.databox(test_kwarg='test_value')
-        
-
     def test_h_str(self):
         self.databox.load_file(path=self.data_path3)         
         val = self.databox.h('header1')
-        exp = 'value1'
+        exp = None
         self.assertEqual(val, exp)          
```

Comment:
'header1' has the value 'value1'.  See this fixture:  https://github.com/Spinmob/spinmob/blob/master/tests/fixtures/data_types/headers_xy.dat